skorch/callbacks/lr_scheduler.py

```diff
@@ -87,6 +87,11 @@ def simulate(self, steps, initial_lr):
         initial_lr: float
           Initial learning rate
 
+        step_args: Any
```

Comment:
Would it be more helpful to change the type to `None or float or list of float (default=None)`?

Yes, technically it doesn't have to be a list but I don't think it still makes it easier for users to know what they need to pass.